lib/trino-parquet/src/main/java/io/trino/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -485,6 +485,11 @@ private static Domain getDomain(Type type, DictionaryDescriptor dictionaryDescri
         }
 
         int dictionarySize = dictionaryPage.get().getDictionarySize();
+
+        if (dictionarySize == 0) {
+            return Domain.none(type);
```

Comment:
souldn't it be `return Domain.all(type);` ?

Reply:
Actually, it should be `Domain.onlyNull(...)`, as an empty dictionary indicates a data page that can only contain nulls.